features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx

```diff
@@ -304,6 +304,9 @@ function TryOutController(props) {
         switch (name) {
             case 'selectedEnvironment':
                 setSelectedEnvironment(value, true);
+                // eslint-disable-next-line no-case-declarations
+                const urls = environmentObject.find((elm) => value === elm.environmentName).URLs;
```

Comment:
environmentObject will not be available for the normal TryOut Console so you might have to do this only if the environmentObject is available.